test/choosetests.jl

```diff
@@ -45,11 +49,14 @@ function choosetests(choices = [])
         # specifically selected case
         filter!(x -> x != "linalg", tests)
         prepend!(tests, ["linalg1", "linalg2", "linalg3", "linalg4",
-            "linalg/lapack", "linalg/triangular", "linalg/tridiag",
-            "linalg/bidiag", "linalg/diagonal",
-            "linalg/pinv", "linalg/givens", "linalg/cholesky", "linalg/lu",
-            "linalg/arnoldi", "linalg/symmetric"])
+                         "linalg/lapack", "linalg/triangular", "linalg/tridiag",
+                         "linalg/bidiag", "linalg/diagonal",
+                         "linalg/pinv", "linalg/givens", "linalg/cholesky", "linalg/lu",
+                         "linalg/symmetric"])
+        if Base.USE_GPL_LIBS
+            tests = [tests, "linalg/arnoldi.jl"]
```

Comment:
This also breaks CI due to the `.jl` suffix. And there's an old-style concatenation warning here.